test/decl/operator/declared_power_operator.swift

```diff
@@ -0,0 +1,12 @@
+// RUN: %target-typecheck-verify-swift
+
+infix operator **
+
+func **(a: Double, b: Double) -> Double { return 0 }
+
+func testValidPowerOperation() { 
```

Comment:
```suggestion
func testPowerOperator() { 
```

I would not go so far as to call it a _valid power operation_: it does no such thing :)